### DISCUSSION TITLE
Restore from failed hard drive

### DISCUSSION THREAD

**Discussion by MVerBerkmoes** [OrcaSlicer/OrcaSlicer#5033] [Q&A]
Using OS 1.9.1 on W11.  Had a HD crash and wonder if there is a way to restore printers and filament settings from my backup.  I have the entire _C:\Program Files\OrcaSlicer_ and _C:\Users\<username>\AppData\Roaming\OrcaSlicer_ folders available to restore.

I have run the setup on the new hard drive to create a generic printer and filament setup, but I'd like to simply lay folders from the backup  over the new install and be back up and running with all of my settings.  I'm not excited about reinventing the wheel again if it's not necessary.

I have restored the whole _C:\Users\<username>\AppData\Roaming\OrcaSlice_r folder and the _C:\Program Files\OrcaSlicer\resources\profiles_ folders, but this hasn't worked.  I'm missing something ....

Any suggestions anyone can offer would be greatly appreciate!

**Comment by LJKraus** [comment]
I don't have a sure answer, only a possibility, but you've not gotten any other response, so...

Within C:\Users\<username>\AppData\Roaming\OrcaSlicer\user there is a numbered folder that contains your user presets.

I don't know the source of the number, but maybe the new installation is using a different number than the old.  See if you have two numbers.  If you do, copy the contents of the old folder to the new and restart Orca.
